.github/workflows/main.yml

```diff
@@ -0,0 +1,94 @@
+name: Godot CI
+
+on:
+    pull_request:
+    push:
+      branches: [ master, 3.2, 3.1, 3.0 ]
+env:
+  SCONS_CACHE_MSVC_CONFIG: true
+  SCONS_CACHE_LIMIT: 2048
```

Comment:
For the reference, do you know what's the max cache size for GH actions? Is the cache max size split across all jobs, or is it a max per job?

Reply:
Yes, the cache size is limited per repository to 5GB. It will then throw out the cache, by 'oldest first'.

Godot builds roughly a 400-500 MB cache on windows, so this cache for release and editor is approximately 1GB for the windows platform.

Artifacts are counted separately.